GoogleForCreators/web-stories-wp
# Floating menu: Text color

## Context



## Summary
- Adds color picker to the text element floating menu to choose the text color.


## Relevant Technical Choices



## To-do




## User-facing changes
When there are multiple colors in a text:
<img width="587" alt="Screenshot 2022-03-09 at 13 15 03" src="https://user-images.githubusercontent.com/3294597/157431015-a57c9cfb-c1ca-4f76-925d-d35ebbc973dc.png">

When there is single color in the text
<img width="540" alt="Screenshot 2022-03-09 at 13 14 18" src="https://user-images.githubusercontent.com/3294597/157430950-c3fe2c20-dd77-4076-8687-2d0ecc25a1ed.png">




## Testing Instructions






This PR can be tested by following these steps:

1. Add a text
2. Verify you can assign color to the text from the floating menu
3. Verify the eyedropper also works

Note: general known bug: eyedropper doesn't close unless the user chooses a color.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?
Tracks when color is set from the menu:
```
trackEvent('floating_menu', {
        name: 'set_text_color',
        element: 'text',
      });
```


### Does this PR have a legal-related impact?
No


## Checklist




---



Fixes #10632
